cachito/workers/config.py

```diff
@@ -3,16 +3,41 @@
 import tempfile
 import logging
 
+import kombu
+
 from cachito.errors import ConfigError
 
 
 class Config(object):
     """The base Cachito Celery configuration."""
 
-    # Don't use the default 'celery' queue
-    task_default_queue = 'cachito'
     cachito_auth_type = None
     cachito_log_level = 'INFO'
+    # The task messages will be acknowledged after the task has been executed,
+    # instead of just before
+    task_acks_late = True
```

Comment:
Some tasks may take a while to complete. If we have more than worker for a given queue, won't this cause messages to be resent while actually being processed? 

Even if we set `worker_concurrency` to 1, as soon as we have more than one worker pod, we may hit this situation.

Reply:
@lcarva why do you think it will be resent if it takes too long? The heartbeat of the Celery worker will be kept alive with RabbitMQ during this time.

Reply:
Something to watch out for. I'm not sure if the heartbeat will be maintained. I was seeing some odd behavior when using Cachito to fetch the sources for kubernetes.

Reply:
Was it using this configuration or the configuration before this PR?

Reply:
Before this PR.